### PR TITLE
Welcome/Help Menu

### PR DESCRIPTION
**Pull request by kelseyroy**
Adds:
- Menu to display main menu with options to start a new game, usage/instructions, or allows the player to quit
- Play that now contains the gameplay functionality
- More messages to the player, including messages announcing the results of the game and asking a player if they would like to play again or quit.

Refactors:
- Program to display menu, and adds a finally statement that allows the game to gracefully quit.

UI changes:
Screenshot 2023-05-17 at 4 00 27 PM

Screenshot 2023-05-17 at 4 00 41 PM

Screenshot 2023-05-17 at 3 55 33 PM

https://github.com/kelseyroy/wordle/assets/89801075/3184c23f-7043-49a2-9f92-1bcca77f2bb7

### CODE REVIEW

File: Wordle.UI/ConsoleUI.cs [hunk 1]
```diff
@@ -81,6 +81,7 @@ private void RenderRow(WordScore word)
     }
     private void RenderCell(LetterScore letter)
     {
+        Console.ForegroundColor = ConsoleColor.Black;
```
Comment (t-keazirian) [inline Wordle.UI/ConsoleUI.cs:84]: Nice - this makes it so much easier to see the letters!

File: Wordle.UI/Menu.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    }
+    private void Logo(ConsoleColor foreground, ConsoleColor background)
+    {
+        ConsoleColor[] colors = new ConsoleColor[] { ConsoleColor.DarkGreen, ConsoleColor.DarkGray, ConsoleColor.DarkYellow };
+        Console.WriteLine("Welcome to" + Environment.NewLine);
+        Console.WriteLine("╔═══╦═══╦═══╦═══╦═══╦═══╗");
+        foreach (char letter in "WORDLE")
+        {
+            Console.Write("║");
+            Random random = new Random();
```
Comment (t-keazirian) [inline Wordle.UI/Menu.cs:43]: This is cool! I tried a few incorrect menu options and it changed the Logo at the top - didn't expect that!

**Review by t-keazirian — COMMENTED** [review]
Noticed a bug during testing:
Screenshot 2023-05-18 at 8 36 55 AM
Screenshot 2023-05-18 at 8 50 45 AM

Screenshot 2023-05-18 at 8 37 45 AM

It seems like the logic is a little off somehow - that when if letter is present in the word, it's putting green where it is and then yellow everywhere else. Thoughts?

**Comment by kelseyroy** [comment]
> It seems like the logic is a little off somehow - that when if letter is present in the word, it's putting green where it is and then yellow everywhere else. Thoughts?

@t-keazirian This will hopefully be fixed in the next PR you see! This is the logic that I'm trying to address with the duplicate letters ticket I'm working on. Honestly I have a little bug in duplicate letters at the moment, which is why I haven't pushed it up yet with the latest PR. But coming soon!

**Review by t-keazirian — APPROVED** [review]
Looks great!!  Awesome refactor with the `Play` class to reduce the code in `Program`!

**Pull request merged**